docs/compile.md

```diff
@@ -188,7 +188,69 @@ It's up to you to decide what the output is.
 
 ### kadet
 
-This input type is experimental. See <https://github.com/deepmind/kapitan/pull/190> for its usage.
+Kadet is an extensible input type that enables you to generate templates using python. The key benefit being the ability to utilize familiar programing principles while having access to kapitan's powerful inventory system. 
+
+A library that defines resources as classes using the Base Object class (BaseObj) is required. These can then be utilized within components to render output. 
+
+The following functions are provided by the class BaseObj.
+
+Method definitions:
+
+* new: Provides parameter checking capabilities
```

Comment:
Maybe update method names to `function()`? So we'd have `new()`, `body()`, etc...

Reply:
True, that does make it more clear. I change references to BaseObj to reflect this also. This just helps readers better understand the meaning. 

I removed the following reference on line 193 since it generally did not feel right.

- Base Object class (`BaseObj()`) is required